.github/workflows/auto-updates.yaml

```diff
@@ -221,17 +233,28 @@ jobs:
         delete-branch: true
 
         # Description of the change and PR body.
-        commit-message: 'upgrade to latest dependencies'
+        commit-message: |
+          upgrade to latest dependencies
+
+          ${{ steps.updatedeps.outputs.deplog }}
```

Comment:
How does this interact with `release-notes` blocks?

Reply:
It's unrelated and doesn't interact